### DISCUSSION TITLE
[WASM] Strange icon visualization on drag&drop

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#8277] [Q&A]
I implemented a custom TabView where I could drag&drop the tabs which are simple borders. However, there is a strange rectangle shown underneath the (correct) MOVE icon when dragging the tab header/border. I attached a brief video.

Any idea what is going on there?

https://user-images.githubusercontent.com/5839577/157452369-1db97993-1116-4849-a3c4-08e2a04554e5.mp4

**Comment by dr1rrb** [comment]
Did you tried with Firefox? I suspect this border is injected by the browser :/

**Reply by DierkDroth** [reply]
@dr1rrb thanks for follow up. How could I try Firefox from VisualStudio (DEBUG)?

**Reply by jeromelaban** [reply]
Open the same url in a different browser, or change the active browser in the debug selector:

![image](https://user-images.githubusercontent.com/5839577/157467495-25b7c081-0887-4e5f-b60e-ee2694dc8dba.png)

**Reply by DierkDroth** [reply]
>Open the same url in a different browser

Sorry @jeromelaban , how would this work when starting from  VS in DEBUG mode?

> active browser in the debug selector:

Where  would I find that 'debug selector' in VS? Is it in a toolbar? Which one? Else?

**Reply by jeromelaban** [reply]
You don't need debug anything there, just start the app to see if the behavior is different with Firefox.

**Reply by DierkDroth** [reply]
@jeromelaban I'm probably too dump to understand what you wanted me to do. 

I just press F5 in VS and Chrome + UNO WASM comes up (DEBUG mode). How could I use FireFox instead of Chrome?

**Reply by dr1rrb** [reply]
Well I just want to confirm if that border is injected by the browser (which often allow to drag and drop almost any element in a page) or if it's uno which does create it for an invalid reason (like trying to inject a screenshot of the element which is not feasible on wasm). Firefox has would most probably have different behavior there.

**Reply by DierkDroth** [reply]
@dr1rrb absolutely! 

I simply don't know how to use FireFox instead of Chrome in my DEBUG scenario above. I just needed instructions to fill that gap. Thanks in advance

**Reply by jeromelaban** [reply]
> @jeromelaban I'm probably too dump to understand what you wanted me to do.
> 
> I just press F5 in VS and Chrome + UNO WASM comes up (DEBUG mode). How could I use FireFox instead of Chrome?

You have a app which deploys to a local url, open that url in Firefox.

**Reply by dr1rrb** [reply]
And to test it, start the app as usual with chrome, then while it's running (keep Chrome open) open Firefox and go to http://localhost:59847

**Reply by DierkDroth** [reply]
@dr1rrb thanks for guidance! That helped

OK, so FireFox shows the same strange rectangle underneath the MOVE icon

**Reply by DierkDroth** [reply]
... and since we're there: same odd behavior on Edge

**Reply by DierkDroth** [reply]
@jeromelaban what could be done to address the issue?

**Reply by jeromelaban** [reply]
Let's track this in https://github.com/unoplatform/uno/issues/8307

**Reply by DierkDroth** [reply]
@jeromelaban I just retested this issue. Appears it no longer exists (on Chrome). 

I suggest you closed #8307 if that made sense to you. Thanks
